src/register_target.cpp

```diff
@@ -44,8 +66,6 @@ std::unordered_map<std::string, target>& target_map()
     return m;
 }
 
-void register_target_init() { (void)target_map(); }
```

Comment:
Dont remove this, I think this was necessary to ensure targets are loaded in some cases.